components/builder-web/stylesheets/_dialog.scss

```diff
@@ -30,3 +30,8 @@
     }
   }
 }
+
+// Angular Material overrides
+.cdk-overlay-dark-backdrop {
+  background: $hab-blue-dark;
+}
```

Comment:
For dialogs, the transparent overlay is now based upon the hab dark blue color as opposed to the default Angular Material black/dark gray.